packages/live-status-gateway/src/wsHandler.ts

```diff
@@ -34,15 +34,22 @@ export abstract class WebSocketTopicBase {
 		this._logger.error(`Process ${this._name} message not expected '${JSON.stringify(msg)}'`)
 	}
 
-	sendMessage(ws: WebSocket, msg: object): void {
+	sendMessage(recipients: WebSocket | Iterable<WebSocket>, msg: object): void {
 		const msgStr = JSON.stringify(msg)
-		this._logger.debug(`Send ${this._name} message '${msgStr}'`)
-		ws.send(msgStr)
+
+		recipients = isIterable(recipients) ? recipients : [recipients]
+
+		let count = 0
+		for (const ws of recipients) {
+			count++
+			ws.send(msgStr)
+		}
+		this._logger.silly(`Send ${this._name} message '${msgStr}' to ${count} recipients`)
 	}
 
 	sendHeartbeat(ws: WebSocket): void {
 		const msgStr = JSON.stringify({ event: 'heartbeat' })
-		this._logger.silly(`Send ${this._name} message '${msgStr}'`)
+		// this._logger.silly(`Send ${this._name} message '${msgStr}'`)
```

Comment:
Should this be removed? Or just kept, but easier to filter out?

Reply:
I think we should remove it, unless we have any specific issue that we need to be able to troubleshoot in prod.